src/collections/BUILD

```diff
@@ -1,5 +1,5 @@
 cc_library(
-    name = "vector",
-    hdrs = ["vector.h"],
-    visibility = ["//visibility:public"],
-    )
+	name = "vector",
```

Comment:
why use tab instead?

Reply:
I get used to it and I think it necessary to standardize it with a specific standard.